### PR TITLE
populate RunService and SampleService

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql [hunk 1]
```diff
@@ -886,12 +909,12 @@ DELIMITER ;
 
 DROP VIEW IF EXISTS CompletedPartitions;
 CREATE OR REPLACE VIEW RunPartitionsByHealth AS
-  SELECT pool_poolId AS poolId, sequencingParameters_parametersId as parametersId, COUNT(*) AS num_partitions, health AS health, MAX((SELECT MAX(changeTime) FROM RunChangeLog WHERE RunChangeLog.runId = Run.runId)) as lastUpdated
```
Comment (apmasell) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql]: This reverts my fix for the order completions.

File: miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/RunService.java [hunk 2]
```diff
@@ -1,8 +1,87 @@
...
 
+  Collection<Run> list() throws IOException;
+
+  Collection<Run> listWithLimit(long limit) throws IOException;
+
+  int count() throws IOException;
+
+  Map<String, Integer> getRunColumnSizes() throws IOException;
+
+  Collection<Run> listBySearch(String query) throws IOException;
```
Comment (apmasell) [inline miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/RunService.java]: Do we need these (and the sample versions)? Can we use the pagination interface to do that? We might need sequencer and container, but I think the rest are covered.

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql (hunk 1 above)
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql]: Weird. I'll fix that.

File: miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/RunService.java (hunk 2 above)
Comment (callunity) [inline miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/RunService.java]: I'll make some updates to that in another PR.

**Pull request merged**